### PR TITLE
label_sync: add a grooming label

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

/kind misc

/cc @nikhil-thomas @pradeepitm12 @abayer @afrittoli 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

**Comment by vdemeester** [comment]
Also adds platform labels 👼🏼

**Review by imjasonh — COMMENTED** [review]
/lgtm

File: label_sync/labels.yaml [hunk 1]
```diff
@@ -353,6 +353,23 @@ repos:
...
+        name: grooming
+        target: issues
+        addedBy: humans
+      - color: 0c6a89
+        description: This issue targets the OpenShift Platform
+        name: platform/openshift
+        target: issues
+        addedBy: humans
+      - color: 0c6a89
+        description: This issue is used during working group to mark issues to be discussed
```
Comment (abayer) [inline label_sync/labels.yaml:369]: The description for this and `grooming` are the same - I'm assuming this one should be something else?

**Pull request merged**